README.md

```diff
@@ -103,3 +103,33 @@ By default, The Lounge will run using the `node (1000:1000)` user. This is custo
 Beware that this may cause permission issues when a container process tries reading from the data disk unless you have manually set the permissions correctly.
 
 Also keep in mind that whenever executing one-off commands in the container you need to explicitly set the correct user.
+
+### Image building (advanced usage)
+
+If you would like to build your own Docker image, you can do so with the following command:
+
+```sh
+docker build -t thelounge:custom .
+```
+
+On some older hardware, you may encounter build errors if the current version of one or more dependencies does not publish pre-compiled binaries for your architecture anymore. The build process can compile the binaries from source, but you will need to install the necessary build dependencies first. To do this, you can make the following changes to your local copy of the Dockerfile:
+
+- Add additional required packages to the `apk add` command. This normally includes `python3`, `make`, and `build-base`.
+- Link any dependencies as necessary. This normally involves creating a symlink for `python` that points at `python3`.
```

Comment:
iirc that shouldn't be needed on the current master as the dep that did that updated.

Reply:
When I build using version 4.3.1, some part of the `node-sqlite3` module build process is still expecting Python to be `python`. I can't test against master because the Dockerfile pulls from the yarn package, but I reworded things to suggest it as something you should only do if you encounter an error, just in case the issue persists or comes back in the future.

Can still remove it entirely, if you'd prefer.